torch/_inductor/kernel/conv.py

```diff
@@ -417,4 +432,5 @@ def _convolution(
     )
 
 
-add_layout_constraint(aten.convolution, constrain_to_fx_strides)
+if not config.layout_opt:
+    add_layout_constraint(aten.convolution, constrain_to_fx_strides)
```

Comment:
Looking at config flags in global scope won't pick up dynamic changes.

Reply:
yea, I added the constraint back later if layout_opt is dynamically disabled. I'll find a better way to handle this